_posts/programme/2021-03-24-finale.md

```diff
@@ -41,6 +41,9 @@ All times in UTC.
 - **16:05**: The International Council of RSE Associations
 
   Stephan Druskat, _German Aerospace Center (DLR), Humboldt-Universität zu Berlin, Friedrich Schiller University Jena_
+
+  > **Abstract:** Ever since the UK RSE association was founded in [2013](https://society-rse.org/about/history/), the number of national RSE associations and initiatives has steadily increased, and there are now groups in [Germany](https://de-rse.org), [the Netherlands](https://nl-rse.org/), [Africa](https://rsse.africa/), [the Nordics](https://nordic-rse.org/), [Belgium](https://www.be-rse.org/), the [United States](https://us-rse.org/), and the [Society of Research Software Engineering](https://society-rse.org) in the UK. Informally, international collaboration between these groups has a good and long tradition, be it serving on committees for each others' national conferences, running a concerted RSE survey, or this very series of online research software events. With the [recently founded International Council of RSE Associations](https://researchsoftware.org/2021/01/27/introducing-the-international-council-of-RSE-associations.html), there is now an open forum for national and multinational RSE associations to talk and coordinate regularly, and thus sustain this international collaboration. In this talk, I will briefly introduce the Council, and its aims and setup.
```

Comment:
The UK RSE association was founded in [2013](https://society-rse.org/about/history/), and since then, additional national RSE associations and initiatives have formed: there are now groups in [Germany](https://de-rse.org), [the Netherlands](https://nl-rse.org/), [Africa](https://rsse.africa/), [the Nordics](https://nordic-rse.org/), [Belgium](https://www.be-rse.org/), and the [United States](https://us-rse.org/), along with the [Society of Research Software Engineering](https://society-rse.org) in the UK. These international groups informally talk and collaborate in a variety of settings, be it serving on committees for each others' national conferences, running a concerted RSE survey, or this very series of online research software events. With the [recently founded International Council of RSE Associations](https://researchsoftware.org/2021/01/27/introducing-the-international-council-of-RSE-associations.html), there is now an open forum for national and multinational RSE associations to talk and coordinate regularly, and thus sustain this international collaboration. In this talk, I will briefly introduce the Council, and its aims and setup.